src/Language/PureScript/Renamer.hs

```diff
@@ -175,7 +177,7 @@ renameInLiteral _ l = return l
 -- Renames within case alternatives.
 --
 renameInCaseAlternative :: CaseAlternative Ann -> Rename (CaseAlternative Ann)
-renameInCaseAlternative (CaseAlternative bs v) =
+renameInCaseAlternative (CaseAlternative bs v) = newScope $
```

Comment:
The changes in this module are just leftovers which I can revert for the most part, but I think this needs to be here, yes?